test/Test/Pos/Ssc/GodTossing/SeedSpec.hs

```diff
@@ -43,23 +45,23 @@ spec = do
     describe "calculateSeed" $ smaller $ do
         prop
             "finds the seed when all openings are present" $
-            do n <- sized $ \size -> choose (1, max size 1)
+            do n <- sized $ \size -> choose (4, max size 4)
                return $ recoverSecretsProp n n 0 0
         prop
             "finds the seed when all shares are present" $
-            do n <- sized $ \size -> choose (1, max size 1)
+            do n <- sized $ \size -> choose (4, max size 4)
                return $ recoverSecretsProp n 0 n 0
         prop
             "finds the seed when all secrets can be recovered" $
-            do n <- sized $ \size -> choose (1, max size 1)
+            do n <- sized $ \size -> choose (4, max size 4)
```

Comment:
SCRAPE doesn't work when there are less than 4 participants, because it requires the threshold to be between 2 and N−2 (N = number of participants).